.github/workflows/pr-labeler.yml

```diff
@@ -3,7 +3,9 @@ name: Pull Request Labeler
 
 # yamllint disable-line rule:truthy
 on:
-  - pull_request_target
+  # zizmor: ignore[dangerous-triggers] This is safe because we are not running any untrusted code
```

Comment:
It is probably safer, but I wonder if we should pin the `- uses: actions/labeler@v5` too.

Reply:
Yes we should

>Official GitHub actions namespaces can be pinned by branch or tag. In other words, actions/checkout@v4 is acceptable, but actions/checkout is not.

They are excluded by default so they are not reported but I can pin them anyway